packages/jupyter-ai/src/chat_handler.ts

```diff
@@ -164,6 +140,41 @@ export class ChatHandler implements IDisposable {
     (value: AiService.AgentChatMessage) => void
   > = {};
 
+  private _onClose(reject: any) {
+    reject(new Error("Chat UI websocket disconnected"))
+    console.error("Chat UI websocket disconnected")
```

Comment:
Should we have a `console.error` here, and if so, should we indicate that we're trying to reconnect?

Reply:
This is purely for debugging. It will be useful in situations where connection is broken. Agree, that we can add another statement here to indicate that we will try to reconnect.